controllers/api/eventRoutes.js

```diff
@@ -1,38 +1,176 @@
 const router = require('express').Router();
-const { Event } = require('../../models');
+const { Event, User, Task } = require('../../models');
+const sequelize = require('../../config/connection');
 const withAuth = require('../../utils/auth');
 
-router.post('/', withAuth, async (req, res) => {
-  try {
-    const newEvent = await Event.create({
-      ...req.body,
-      user_id: req.session.user_id,
+// get all users
+router.get('/', (req, res) => {
```

Comment:
For the userRoutes I comment that it looks like all the get routes go in the homeRoutes file.  I have posed the question to Ross and Jonathan.  See my comment for the userRoutes for more detail.